config/config_helpers.go

```diff
@@ -318,22 +310,43 @@ var listQuotedParamRegex = regexp.MustCompile(`^"([^"]*?)"\s*,\s*(.*)$`)
 // foo("a", "b") -> return "a", "b", 2, nil
 //
 func parseOptionalQuotedParam(parameters string) (string, string, int, error) {
+	parsed, err := splitParamList(parameters)
+
+	if err != nil {
+		return "", "", 0, err
+	}
+
+	if len(parsed) == 0 {
+		return "", "", 0, nil
+	} else if len(parsed) == 1 {
+		return parsed[0], "", 1, nil
+	} else if len(parsed) == 2 {
+		return parsed[0], parsed[1], 2, nil
+	}
```

Comment:
Otherwise, we're just managing the return value based on the number of parameters we extracted...